cl/alerts/tasks.py

```diff
@@ -100,6 +117,26 @@ def send_docket_alert(d_pk: int, since: datetime) -> None:
     connection = get_connection()
     connection.send_messages(messages)
 
+    # Notify all the webhooks available for the users subscribed
+    for user in users:
```

Comment:
OK, so sorry to take so long getting back to this. My schedule is returning to normal, finally. 

I think this PR is good, but my fresh eyes did, unfortunately, catch something. In your code here...

    for user in users

...you're determining who to send webhooks to based on who is subscribed for email alerts on a docket (`users` is being set by `User.objects.filter(docket_alerts__docket_id=d_pk).distinct()`, above). 

That's not right. Webhooks should get sent based on the account that received the email. So if Jenny is the one that set up her PACER account to forward emails to jenny@recap.email, then it's her account that should be getting the webhook POST.

Reply:
This issue makes me think that sending webhooks should be a second function, since we only want to send them from `process_recap_email`, not every time we do alerts — at least not for now. Eventually, I think we'll want webhooks that can be set up to receive regular alert emails, but these webhooks are just for when your account gets a recap.email email.